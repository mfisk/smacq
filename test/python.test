#! /bin/sh

TESTFILE=$TEST/0325@1112-snort.pcap
PYTHONPATH=$TEST; export PYTHONPATH

${BINDIR}smacqq -m <<EOF 
pcapfile $TESTFILE | python("pythontest.Dumper", "foobar") | print ts
EOF

exit 1
