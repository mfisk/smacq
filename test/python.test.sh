#! /bin/sh

TESTFILE=0325@1112-snort.pcap
PYTHONPATH=$TEST; export PYTHONPATH

$SMACQQ -f - <<EOF 
pcapfile $TESTFILE | python("pythontest.Dumper", "foobar") | print ts
EOF

exit 1
