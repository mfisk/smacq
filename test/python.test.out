('init', ('foobar',))
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 93L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 97L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
954011728.020237
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 126L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 191L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 346L)
954011728.028445
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
954011728.129182
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
954011728.620389
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
954011729.019047
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
954011729.119403
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
954011731.393278
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 126L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 191L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 256L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 158L)
954011731.434037
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 117L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 246L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 99L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 154L)
954011731.449118
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 256L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 146L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 154L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 117L)
954011731.479025
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 222L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 99L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 154L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 117L)
954011731.485619
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 218L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 206L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 236L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 218L)
954011731.491501
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 198L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
954011731.527459
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
954011731.533792
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
954011731.539540
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
954011731.545646
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
954011731.552164
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
954011731.768626
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 126L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 210L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 99L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 97L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
954011731.914011
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 93L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x11')
('len', 'uint32', <type 'long'>, 92L)
('srcip', 'ip', <type 'str'>, '192.168.0.200')
('dstip', 'ip', <type 'str'>, '192.168.0.54')
('ipprotocol', 'ipproto', <type 'str'>, '\x11')
('len', 'uint32', <type 'long'>, 307L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 346L)
954011731.945492
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 346L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 330L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 330L)
954011747.127418
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('del', 3)
